tests/test_A1_csv_to_json.py

```diff
@@ -373,3 +373,67 @@ def test_add_storage_components_label_correctly_added():
 def teardown_function():
     if os.path.exists(os.path.join(CSV_PATH, CSV_FNAME)):
         os.remove(os.path.join(CSV_PATH, CSV_FNAME))
+
+
+def test_default_values_storage_without_thermal_losses():
+    exp = {
+        "fixed_thermal_losses_relative": {UNIT: "no_unit", VALUE: 0},
+        "fixed_thermal_losses_absolute": {UNIT: "kWh", VALUE: 0},
+    }
+
+    data_path = os.path.join(
+        TEST_REPO_PATH,
+        "benchmark_test_inputs",
+        "Feature_stratified_thermal_storage",
+        "csv_elements",
+    )
+
+    json = A1.create_json_from_csv(
+        input_directory=data_path,
+        filename="storage_fix_without_fixed_thermal_losses",
+        parameters=[
+            "age_installed",
+            "development_costs",
+            "specific_costs",
+            "efficiency",
+            "installedCap",
+            "lifetime",
+            "specific_costs_om",
+            "unit",
+        ],
+        asset_is_a_storage=True,
+    )
+    for param, result in exp.items():
+        assert json["storage capacity"][param][VALUE] == result[VALUE]
+
+
+def test_default_values_storage_with_thermal_losses():
```

Comment:
are you asserting somewhere else (eg. in the benchmark) that the losses can also be a timeseries?

Please add the test to the list of functions that tests `A1.create_json_from_csv()` (comp "Notes" section of other functions).

Reply:
> Please add the test to the list of functions that tests `A1.create_json_from_csv()` (comp "Notes" section of other functions).

Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/485bd76f14a83992cc29bfdc62a8bb6ef728eb64


> are you asserting somewhere else (eg. in the benchmark) that the losses can also be a timeseries?

Yes, exactly. I'm checking this in `test_benchmark_stratified_thermal_storage.py`